### PR TITLE
pgbouncer deployment without vault and with vault done.

### PR DESCRIPTION
**Pull request by stephenstubbs**

### CODE REVIEW

File: experimental/pgbouncer/helmfile.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# generate a transiant postgres-xl db. This db will 
+# NOT be able to withstand restarts.
+releases:
+  - name: zav-db-tran
```
Comment (LamaAni) [inline experimental/pgbouncer/helmfile.yaml]: Can you change this to a more descriptive name?

File: experimental/pgbouncer/pwd_secret.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  pgxl-passwords-collection
+type: Opaque
+data:
+  # You must base64 encode your values. See: https://kubernetes.io/docs/concepts/configuration/secret/
+  pgpass: "eW91cl9wYXNzd29yZDE="
```
Comment (LamaAni) [inline experimental/pgbouncer/pwd_secret.yaml]: Please remove specific values (and especially passwords). You can keep this in a local fork- though I would suggest not putting passwords in a git repo at all.

**Review by LamaAni — CHANGES_REQUESTED** [review]

File: experimental/pgbouncer/pwd_secret.yaml (hunk 2 above)
Comment (stephenstubbs) [reply experimental/pgbouncer/pwd_secret.yaml]: ok sure this is just your_password1 which is in the install.sh as an example that people can change to their needs but I understand and will remove it.

File: experimental/pgbouncer/helmfile.yaml (hunk 1 above)
Comment (stephenstubbs) [reply experimental/pgbouncer/helmfile.yaml]: ok done

File: experimental/pgbouncer/pwd_secret.yaml (hunk 2 above)
Comment (LamaAni) [reply experimental/pgbouncer/pwd_secret.yaml]: You are correct. I will do the same and remove it from the other location.

**Review by LamaAni — APPROVED** [review]

**Pull request merged**